site/src/modules/resources/AgentRow.test.tsx

```diff
@@ -106,3 +107,60 @@ describe.each<{
 		}
 	});
 });
+
+describe.each<{
+	props: Partial<AgentRowProps>;
+}>([
+	{
+		props: {
+			agent: {
+				...MockWorkspaceAgent,
+				apps: [
+					{
+						...MockWorkspaceApp,
+						hidden: false,
+					},
+				],
+			},
+		},
+	},
+	{
+		props: {
+			agent: {
+				...MockWorkspaceAgent,
+				apps: [
+					{
+						...MockWorkspaceApp,
+						hidden: true,
+					},
+				],
```

Comment:
curious if you can define one agent with 2 apps, one hidden and visible 🤔

Reply:
That's a good test actually.

Reply:
Yeah that's probably a better idea, to make sure one being hidden doesn't accidentally hide them all.